vis/xAPISG-ItemsActionTypeInteracted.ipynb

```diff
@@ -0,0 +1,84 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "%run vis/xAPISG-noDataToFillVisualization.ipynb # notebook to create the visualisation with a message NoDataToFill\n",
+    "%run vis/HelpersFunctions/MultiBarMultiColor.ipynb\n",
```

Comment:
Raises error "ERROR:root:File `'vis/HelpersFunctions/MultiBarMultiColor.ipynb.py'` not found." when running analysis on xAPISGProcessor notebook.